hs_core/management/commands/debug_composite_resource.py

```diff
@@ -42,6 +44,7 @@ def debug_resource(short_id):
                     print("    {} does not exist".format(f.storage_path + "_resmap.xml"))
                 if not storage.exists(f.storage_path + "_meta.xml"):
                     print("    {} does not exist".format(f.storage_path + "_meta.xml"))
+         '''
```

Comment:
Why adding lines 33 and 47 to make the enclosed statements a string? Line 35 looks like a comment. Should these lines be removed all together?